server.go

```diff
@@ -1719,10 +1719,10 @@ func (s *server) findPeerByPubStr(pubStr string) (*peer, error) {
 // the cleanup routine to exit early.
 //
 // NOTE: This MUST be launched as a goroutine.
-func (s *server) peerTerminationWatcher(p *peer) {
+func (s *server) peerTerminationWatcher(p *peer, ready chan struct{}) {
```

Comment:
`ready` should be explained in the godoc

Reply:
fixed!